airbyte-cdk/python/unit_tests/sources/streams/concurrent/scenarios/thread_based_concurrent_stream_scenarios.py

```diff
@@ -30,7 +31,8 @@
     primary_key=[],
     cursor_field=None,
     logger=logging.getLogger("test_logger"),
-    cursor=NoopCursor(),
+    message_repository=_message_repository,
+    cursor=None,
```

Comment:
When we don't provide a cursor, the `DefaultStream` behavior has been changed to instantiate the `FinalStateCursor`

Reply:
(previously the DefaultStream would instantiate a NoOpCursor, so we could have/should have made this change before)